api/FileList.json

```diff
@@ -9,7 +9,7 @@
         ],
         "support": {
           "chrome": {
-            "version_added": "1"
+            "version_added": "2"
```

Comment:
The data below says Safari 4, so that's consistent with this not being available in WebKit before the first release of Chrome.